thinc/layers/reduce_first.py

```diff
@@ -20,7 +21,10 @@ def forward(model: Model[Ragged, OutT], Xr: Ragged, is_train: bool) -> Tuple[Out
     x_shape = Xr.dataXd.shape
     lengths = Xr.lengths
 
+    ainfo = ArrayInfo.from_array(Y)
```

Comment:
Should we just call this variable `array_info` for the sake of clarity? It's not that much longer...

Reply:
Sure, I'll change it.